Text/TextAnalyticsForHealth/TextAnalyticsForHealth.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <AzureFunctionsVersion>v3</AzureFunctionsVersion>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Azure.AI.TextAnalytics" Version="5.1.0-beta.7" />
+    <PackageReference Include="Microsoft.NET.Sdk.Functions" Version="3.0.13" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\Common\Common.csproj" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Update="host.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Update="local.settings.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <CopyToPublishDirectory>Never</CopyToPublishDirectory>
+    </None>
```

Comment:
We should probably check in the local.settings.json file if we want to include it here (i.e. if it is useful for local debugging, include it with the needed environment variable keys but with empty values), or otherwise remove it from the csproj

Reply:
I am very unsure of what the `.csproj` file represents or how it was changed. I think this was changed by VS. I'm happy to change it to whatever you think it best.

Reply:
I would say just go ahead and commit the local.settings.json file, and then you should be good. Like I said it can be useful for local debugging anyways.

Reply:
Oh huh, weird. @bleroy we should probably look into why other csprojs still reference it. But if they all still work then that is fine for you to leave as is for now Jonathan.